ui/src/app/utils/authentication.tsx

```diff
@@ -143,6 +143,28 @@ const getAuthInstance = () => {
   return gapi.auth2.getAuthInstance();
 };
 
+export const getAccessToken = () => {
+  const tokenOverride = window.localStorage.getItem(
+    LOCAL_STORAGE_KEY_TEST_ACCESS_TOKEN
+  );
+
+  if (tokenOverride) {
+    return tokenOverride;
+  } else if (!gapi.auth2) {
+    return null;
```

Comment:
surely we can FP all this (another day)

Reply:
Indeed. This is a code move, but I should follow-up with a small refactor.